flake.nix

```diff
@@ -10,6 +10,10 @@
       flake = false;
     };
     git-hooks.url = "github:cachix/git-hooks.nix";
+    combine-haddock = {
+      url = "https://raw.githubusercontent.com/Plutonomicon/plutarch-plutus/f78b70eca8841a4a009cb6791a59c999cbc68745/nix/combine-haddock.nix";
```

Comment:
Do we need this specific commit? If so, could you add a comment describing why?

Reply:
Yes I will write a comment, we don't _need_ a specific commit, we use a specific commit to avoid being vulnerable to any malicious change in the future.

Reply:
Thinking about it again we don't really need to specify the commit here (sorry, my fault).
Since this is a input for our flake nix automatically add the commit revision to its lock file and in order to update we would have to run `nix flake update`.